generator/osm_source.hpp

```diff
@@ -48,6 +51,17 @@ class EmitterBase
   virtual void GetNames(vector<string> & names) const = 0;
 };
 
+class SyncOfstream
```

Comment:
@syershov Could you please comment it? As far as I remember you're well aware of this part of code.